packages/editor/src/client/blocks/embeds/iframe.tsx

```diff
@@ -6,23 +6,43 @@ export interface IframeEmbedProps {
   title?: string
   width?: number
   height?: number
+  styleHeight?: string
+  styleWidth?: string
+  ratio?: boolean
 }
 
-export function IframeEmbed({url, title, width = 300, height = 300}: IframeEmbedProps) {
-  const ratio = width / height
-
+export function IframeEmbed({
+  url,
+  title,
+  width = 300,
+  height = 300,
+  styleHeight,
+  styleWidth,
+  ratio
+}: IframeEmbedProps) {
+  const ratioVal = width / height
+  const styles = {}
+  if (!ratio) {
+    Object.assign(styles, {
```

Comment:
Why the `Object.assign(...)`

Reply:
Coz it allow you to just delete the style at the embed input and you get rid of the useless default values.

The same logic should be applied to `height`, `width` and `title`.

The Object is updated on input change.

Reply:
due a filter in the editor render FE is done, the object assign can be dropped (no more unexpected behavings on input)